pkg/elevate/elevate_context.go

```diff
@@ -0,0 +1,116 @@
+package elevate
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/openshift/backplane-cli/pkg/utils"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+)
+
+const (
+	elevateExtensionName             = "ElevateContext"
+	elevateExtensionRetentionMinutes = 20
```

Comment:
This may be good candidate for backplane config , default to 20 mins and override from the user values

Reply:
That could be a nice improvement we could make in the future, but which might require further discussions, as we do not want to have it too big.

Reply:
I'm personally -1 to making this user-configurable. If we set a time for this it should be consistent across SREs across the fleet.

I would, however, be open to the idea of having a "do not save elevation reason" configuration option that is individually configurable, so that you'd be prompted _every_ time and never persist the elevation reason that someone could opt-into.

I think of this like elevating to SUDO on a machine. The first time it will prompt for your password, and then there is a timeout where it will not prompt you. If you go a bit of time without using `sudo` it will re-prompt for your password again. The default timeout is configured for all SUDOERS on the machine via /etc/sudoers; however an individual user could `sudo -k` each time they run sudo and it would invalidate the timestamp and prompt them again.

Offhand I'm not aware of any use-case where this may make sense, but I can imagine someone has a workflow where saving the elevation reason is not in their own best interest and they'd like to personally turn it off.

Reply:
Agree with you @iamkirkbater, I have added this idea to [OSD-22250](https://issues.redhat.com/browse/OSD-22250) comments for future improvements...
TY